compal/__init__.py

```diff
@@ -223,7 +32,7 @@ def __init__(self, ip, key=None, timeout=10):
         self.session.hooks['response'].append(self.token_handler)
 
         LOGGER.debug("Getting initial token")
-        self.initial_res = self.get('/')
+        self.initial_res = self.get('/', allow_redirects=False)
```

Comment:
Does the check at line 37 still work? We could also set a low number of [max redirects](http://www.python-requests.org/en/latest/api/#requests.Session.max_redirects)